src/main/java/net/fabricmc/loader/transformer/accesswidener/AccessWidener.java

```diff
@@ -151,6 +154,100 @@ public void read(BufferedReader reader, String currentNamespace) throws IOExcept
 		classes.addAll(parentClasses);
 	}
 
+	public void write(StringWriter writer) {
```

Comment:
I think it might be worth moving access widerners out into their own lib, as all this is copyied right out of loom.